src/weekendService.js

```diff
@@ -0,0 +1,22 @@
+import { addDays } from './holidayService';
+
+export const isWeekend = date => date.getDay() === 6 || date.getDay() === 0;
```

Comment:
Thanks to the horrible API design of the Date prototype in JavaScript, the code to perform these types of calculation is quite difficult to read now that we have a lot more business logic that deals with days of week.

I would consider creating an object that will effectively be consumed as enum in a new file `src/utils/constants.js` like so
```javascript
export const Days = {
  MONDAY: 1,
  TUESDAY: 2,
  WEDNESDAY: 3,
  THURSDAY: 4,
  FRIDAY: 5,
  SATURDAY: 6,
  SUNDAY: 0
};
```
and throughout the application, use this when doing anything with days of the week
e.g.
```javascript
export const isWeekend = date => date.getDay() === Days.SATURDAY || date.getDay() === Days.SUNDAY;
```

Reply:
Great call 👍